README.md

```diff
@@ -390,4 +477,51 @@ And the last step before deploying to production is to generate Solidity verifie
 
 And we are ready to go into production!
 
+### Production
+
+Deployed [`semaphore-mtb`](https://github.com/worldcoin/semaphore-mtb/) verifier contracts and their corresponding verifying keys can be found here:
+
+- Batch size 10, tree depth 30: [Etherscan](https://etherscan.io/address/0x6e37bAB9d23bd8Bdb42b773C58ae43C6De43A590#code)
+- Batch size 100, tree depth 30: [Etherscan](https://etherscan.io/address/0x03ad26786469c1F12595B0309d151FE928db6c4D#code)
+- Batch size 1000, tree depth 30: [Etherscan](https://etherscan.io/address/0xf07d3efadD82A1F0b4C5Cc3476806d9a170147Ba#code)
+
+### Verifying the ceremony
+
+We can verify the keys generated in this ceremony are being used in the production contracts we see above by following the rest of this section.
+
```

Comment:
Missing two important steps:

* Verify that you trust at least one of the participants. (And make it clear you don't need to trust all of them).
* Verify that they actually contributed and you didn't just write their names down. (i.e. link to the github comments)

Reply:
Will explain. Thanks!